week3/Week 3 Playground.playground/Contents.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+// a)
+let range = 0...20
+var nums: [Int] = []
+
+for num in range {
+  nums.append(num)
+}
+for num in nums where num % 2 == 0{
+  print(num)
+}
+print("------")
+
+// b)
+let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"
+let vowels: [Character] = ["a", "A", "e", "E", "i", "I", "o", "O", "u", "U"]
+
+var count = 0
+for char in sentence where vowels.contains(char) {
+  count += 1
+}
+print("'\(sentence)' contains \(count) vowels.")
+print("------")
+
+// c)
```

Comment:
Another way to create array's

```swift
let array1 = Array(0...4)
let array2 = Array(0...4)

for i in array1 {
    for j in array2 {
        print("\(i) x \(j) = \(i * j)")
    }
    print("----------")
}
```

Reply:
Oh cool, I will definitely try to remember to use ranges as needed when creating arrays!